$OpenBSD: patch-src_FbTk_FbTime_cc,v 1.1 2013/01/11 08:11:26 dcoppa Exp $

Compile fix: forgotten parentheses around value
(upstream git commit f587cc9e7e8c40bce778ffc0811a3a6c29107b85)

--- src/FbTk/FbTime.cc.orig	Thu Jan 10 11:32:48 2013
+++ src/FbTk/FbTime.cc	Thu Jan 10 11:36:11 2013
@@ -71,7 +71,7 @@ uint64_t _now() {
         initial = false;
         mach_timebase_info_data_t info;
         if (mach_timebase_info(&info) == 0) {
-            micro_scale *= static_cast<double>info.numer / static_cast<double>(info.denom);
+            micro_scale *= static_cast<double>(info.numer) / static_cast<double>(info.denom);
         }
     }
 
